components/core/src/clp_s/search/clp_search/EncodedVariableInterpreter.cpp

```diff
@@ -34,15 +34,31 @@ bool EncodedVariableInterpreter::encode_and_search_dictionary(
         LogTypeDictionaryEntry::add_double_var(logtype);
         sub_query.add_non_dict_var(encoded_var);
     } else {
-        auto entry = var_dict.get_entry_matching_value(var_str, ignore_case);
-        if (nullptr == entry) {
+        auto entries = var_dict.get_entry_matching_value(var_str, ignore_case);
```

Comment:
We weren't using const in the previous code, but since you are changing the code, let's adpat to our latest coding standard.
```suggestion
        auto const entries = var_dict.get_entry_matching_value(var_str, ignore_case);
```